[2022-10-29 19:00:48] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-29 19:00:48] [ INFO ] [H2DBConnection:19] Cerrando la db...
[2022-10-29 19:01:27] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-29 19:01:28] [ INFO ] [H2DBConnection:19] Cerrando la db...
[2022-10-29 19:07:55] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:07:56] [ INFO ] [H2DBConnection:19] Cerrando la db...
[2022-10-29 19:11:50] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:11:50] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-10-29 19:11:50] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:13:01] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:13:01] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-10-29 19:13:01] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:15:28] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:15:29] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-10-29 19:15:29] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:15:53] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:15:54] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
[2022-10-29 19:15:54] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:16:46] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:16:47] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
[2022-10-29 19:16:47] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:18:30] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:18:31] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
[2022-10-29 19:18:31] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:19:11] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:19:12] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
[2022-10-29 19:19:12] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:19:26] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:19:27] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at Main.main(Main.java:12)
[2022-10-29 19:19:27] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:19:44] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:19:44] [ INFO ] [H2DBConnection:19] Cerrando la db...
[2022-10-29 19:21:31] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:21:32] [ ERROR] [H2DBConnection:21] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:18)
[2022-10-29 19:22:06] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:22:06] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:22:06] [ ERROR] [H2DBConnection:22] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:19)
[2022-10-29 19:22:34] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:22:34] [ ERROR] [H2DBConnection:22] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:19)
[2022-10-29 19:22:34] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:23:02] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:26:33] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-29 19:26:34] [ ERROR] [H2DBConnection:31] Error al conectar a la base de datos.
[2022-10-29 19:27:04] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-29 19:27:04] [ ERROR] [H2DBConnection:31] Error al conectar a la base de datos.
[2022-10-29 19:27:04] [ INFO ] [H2DBConnection:19] Cerrando la db...
[2022-10-29 19:27:57] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:27:58] [ INFO ] [H2DBConnection:19] Cerrando la db...
[2022-10-29 19:28:43] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:28:43] [ INFO ] [H2DBConnection:19] Cerrando la db...
[2022-10-29 19:30:01] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:30:01] [ ERROR] [H2DBConnection:22] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:19)
[2022-10-29 19:30:56] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:30:56] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:45)
	at Main.main(Main.java:12)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 19:30:56] [ ERROR] [H2DBConnection:15] Error al conectar la db
[2022-10-29 19:30:56] [ ERROR] [H2DBConnection:22] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:19)
[2022-10-29 19:31:06] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:31:06] [ ERROR] [H2DBConnection:22] Error al cerrar la db
java.lang.NullPointerException
	at Main.main(Main.java:19)
[2022-10-29 19:31:25] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:31:55] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:32:02] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:32:50] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:34:01] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:34:01] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:45)
	at Main.main(Main.java:11)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 19:34:14] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:37:49] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:37:50] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:45)
	at Main.main(Main.java:11)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 19:37:55] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:38:41] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:38:41] [ ERROR] [H2DBConnection:33] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:29)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:45)
	at Main.main(Main.java:11)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 19:38:47] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:39:10] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:40:12] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:40:51] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:40:58] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:41:12] [ INFO ] [H2DBConnection:28] Conectando a la base de datos...
[2022-10-29 19:47:41] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 19:47:41] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:11)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 19:47:52] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 19:47:53] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:11)
[2022-10-29 19:48:13] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 19:48:13] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:11)
[2022-10-29 19:48:21] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 19:48:22] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:11)
[2022-10-29 19:49:10] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 19:49:11] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.validateUserAndPassword(Engine.java:393)
	at org.h2.engine.Engine.createSession(Engine.java:206)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:11)
[2022-10-29 19:49:20] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 19:50:09] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 19:51:15] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:12:37] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:13:56] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:13:56] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:49)
	at Main.main(Main.java:14)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 20:14:04] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:14:49] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:15:28] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:16:30] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:17:13] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:17:13] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:49)
	at Main.main(Main.java:14)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 20:17:18] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:18:24] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:18:53] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:19:22] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:19:58] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:19:59] [ ERROR] [H2DBConnection:34] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:30)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:49)
	at Main.main(Main.java:14)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-29 20:20:04] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:21:36] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:22:06] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 20:22:29] [ INFO ] [H2DBConnection:29] Conectando a la base de datos...
[2022-10-29 22:46:42] [ INFO ] [H2DBConnection:55] La base de datos ha sido creada...
[2022-10-29 23:01:43] [ INFO ] [H2DBConnection:58] La base de datos ha sido creada...
[2022-10-29 23:03:01] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-10-29 23:03:47] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-10-29 23:04:07] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; se esperaba "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"
Syntax error in SQL statement "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"; SQL statement:
RUNSCRIPTs FROM './createSQL/dentalOffice.sql' [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.parsePrepared(Parser.java:866)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.Engine.openSession(Engine.java:278)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
[2022-10-29 23:10:44] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; se esperaba "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"
Syntax error in SQL statement "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"; SQL statement:
RUNSCRIPTs FROM './createSQL/dentalOffice.sql' [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.parsePrepared(Parser.java:866)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.Engine.openSession(Engine.java:278)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
[2022-10-29 23:10:44] [ ERROR] [H2DBConnection:60] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:58)
	at Main.main(Main.java:14)
[2022-10-29 23:11:19] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; se esperaba "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"
Syntax error in SQL statement "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"; SQL statement:
RUNSCRIPTs FROM './createSQL/dentalOffice.sql' [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.parsePrepared(Parser.java:866)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.Engine.openSession(Engine.java:278)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
[2022-10-29 23:11:19] [ ERROR] [H2DBConnection:60] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:58)
	at Main.main(Main.java:14)
[2022-10-29 23:12:04] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; se esperaba "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"
Syntax error in SQL statement "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"; SQL statement:
RUNSCRIPTs FROM './createSQL/dentalOffice.sql' [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.parsePrepared(Parser.java:866)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.Engine.openSession(Engine.java:278)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
[2022-10-29 23:12:04] [ ERROR] [H2DBConnection:60] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:58)
	at Main.main(Main.java:14)
[2022-10-29 23:12:04] [ INFO ] [H2DBConnection:64] La base de datos ha sido creada...
[2022-10-29 23:12:28] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; se esperaba "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"
Syntax error in SQL statement "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"; SQL statement:
RUNSCRIPTs FROM './createSQL/dentalOffice.sql' [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.parsePrepared(Parser.java:866)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.Engine.openSession(Engine.java:278)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
[2022-10-29 23:12:28] [ ERROR] [H2DBConnection:60] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:58)
	at Main.main(Main.java:14)
[2022-10-29 23:13:40] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; se esperaba "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"
Syntax error in SQL statement "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"; SQL statement:
RUNSCRIPTs FROM './createSQL/dentalOffice.sql' [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.parsePrepared(Parser.java:866)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.Engine.openSession(Engine.java:278)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
[2022-10-29 23:13:40] [ ERROR] [H2DBConnection:63] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:57)
	at Main.main(Main.java:14)
[2022-10-29 23:13:55] [ ERROR] [H2DBConnection:53] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; se esperaba "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"
Syntax error in SQL statement "[*]RUNSCRIPTs FROM './createSQL/dentalOffice.sql'"; expected "ROLLBACK, REVOKE, RUNSCRIPT, RELEASE"; SQL statement:
RUNSCRIPTs FROM './createSQL/dentalOffice.sql' [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.parsePrepared(Parser.java:866)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.Engine.openSession(Engine.java:278)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:50)
	at Main.main(Main.java:14)
[2022-10-29 23:13:55] [ ERROR] [H2DBConnection:63] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:57)
	at Main.main(Main.java:14)
[2022-10-29 23:14:18] [ INFO ] [H2DBConnection:59] La base de datos ha sido creada...
[2022-10-30 00:14:14] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:14:14] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:15:55] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:15:55] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:17:10] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:17:10] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:18:13] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:19:25] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:19:26] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:19:34] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:19:53] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:20:04] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:21:10] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:21:40] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:22:17] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:23:28] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:23:28] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:23:35] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:24:58] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:24:58] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:25:08] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:30:08] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:30:08] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:34:52] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:34:53] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.readAll(DentistDaoH2.java:63)
	at Main.main(Main.java:25)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:35:02] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:35:21] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:43:49] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:43:49] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.readAll(DentistDaoH2.java:63)
	at Main.main(Main.java:25)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-10-30 00:43:56] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-10-30 00:43:57] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:18:58] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:18:59] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:20:33] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:20:33] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:21:06] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:21:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:21:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:27:58] [ ERROR] [H2DBConnection:51] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:17)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-11-03 18:27:58] [ ERROR] [H2DBConnection:61] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:55)
	at Main.main(Main.java:17)
[2022-11-03 18:27:58] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:27:58] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-11-03 18:28:15] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:28:15] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:29:06] [ ERROR] [H2DBConnection:51] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:17)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-11-03 18:29:06] [ ERROR] [H2DBConnection:61] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:55)
	at Main.main(Main.java:17)
[2022-11-03 18:29:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:29:06] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.update(DentistDaoH2.java:135)
	at Main.main(Main.java:33)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-11-03 18:30:06] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:30:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:30:46] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:30:46] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:31:01] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:31:01] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:31:48] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:31:48] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:37:01] [ ERROR] [H2DBConnection:51] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:17)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-11-03 18:37:01] [ ERROR] [H2DBConnection:61] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:55)
	at Main.main(Main.java:17)
[2022-11-03 18:37:01] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:37:01] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-11-03 18:37:22] [ ERROR] [H2DBConnection:51] La base de datos no pudo crear.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:48)
	at Main.main(Main.java:17)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 11 more
[2022-11-03 18:37:22] [ ERROR] [H2DBConnection:61] No se logro conecto la base de datos.
java.lang.NullPointerException
	at com.cvargas.dentalOffice.utils.H2DBConnection.createH2DB(H2DBConnection.java:55)
	at Main.main(Main.java:17)
[2022-11-03 18:37:22] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:37:22] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at Main.main(Main.java:22)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 12 more
[2022-11-03 18:37:29] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:37:29] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:37:29] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:37:29] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:37:29] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:38:08] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:38:08] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:38:08] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:38:08] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:46:19] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 18:46:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:46:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:46:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 18:46:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:00:13] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:00:14] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:02:35] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:02:35] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.service.DentistServiceTest.testReadAllDentist(DentistServiceTest.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 80 more
[2022-11-03 19:02:35] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:02:35] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.readAll(DentistDaoH2.java:63)
	at com.cvargas.dentalOffice.service.DentistService.readAll(DentistService.java:28)
	at com.cvargas.dentalOffice.service.DentistServiceTest.testReadAllDentist(DentistServiceTest.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 82 more
[2022-11-03 19:02:54] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:02:54] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:03:32] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:03:33] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:03:45] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 19:03:45] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:03:45] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:03:45] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:03:45] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:07] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:07] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.service.DentistServiceTest.testCrearDentist(DentistServiceTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 80 more
[2022-11-03 19:04:07] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:07] [ ERROR] [H2DBConnection:32] Error al conectar a la base de datos.
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos puede que ya esté siendo utilizada: "C:/Users/Christian Vargas/dental_office.mv.db". Soluciones Posibles: cierre todas las otras conexiones; use el modo server
Database may be already in use: "C:/Users/Christian Vargas/dental_office.mv.db". Possible solutions: close all other connection(s); use the server mode [90020-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.cvargas.dentalOffice.utils.H2DBConnection.getConnection(H2DBConnection.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:28)
	at com.cvargas.dentalOffice.dao.DentistDaoH2.create(DentistDaoH2.java:15)
	at com.cvargas.dentalOffice.service.DentistService.create(DentistService.java:24)
	at com.cvargas.dentalOffice.service.DentistServiceTest.testCrearDentist(DentistServiceTest.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
	at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Christian Vargas/dental_office.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	... 83 more
[2022-11-03 19:04:16] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:17] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:23] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:24] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:37] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:38] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:49] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:04:50] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:05:04] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:05:04] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:12:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:12:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:12:56] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 19:12:56] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:12:56] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:12:56] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:12:56] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:13:16] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:13:16] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:13:17] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:13:17] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:13:17] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:13:17] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:06] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-03 19:14:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:06] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:18] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:18] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:18] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:18] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:18] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-03 19:14:18] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:55] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-07 16:40:55] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:55] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:55] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 16:40:55] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:25:46] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-07 23:25:46] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:25:46] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:25:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:25:47] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:10] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-07 23:26:10] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:10] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:10] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:10] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:26] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-11-07 23:26:26] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:26] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:26] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-11-07 23:26:26] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-12-03 09:01:19] [ INFO ] [H2DBConnection:57] La base de datos ha sido creada...
[2022-12-03 09:01:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-12-03 09:01:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-12-03 09:01:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-12-03 09:01:19] [ INFO ] [H2DBConnection:27] Conectando a la base de datos...
[2022-12-12 19:53:08] [ INFO ] [DentistServiceImpl:30] Creating dentist...
[2022-12-12 19:53:41] [ INFO ] [DentistServiceImpl:30] Creating dentist...
[2022-12-12 19:53:44] [ INFO ] [DentistServiceImpl:48] Searching all the dentists...
[2022-12-12 19:53:49] [ INFO ] [DentistServiceImpl:37] Searching dentist with id= 1
[2022-12-12 19:54:00] [ INFO ] [DentistServiceImpl:62] Updating all dentist data...
[2022-12-12 19:54:02] [ INFO ] [DentistServiceImpl:76] Updating the dentist's first and last name...
[2022-12-12 19:54:04] [ INFO ] [DentistServiceImpl:37] Searching dentist with id= 1
[2022-12-12 19:54:04] [ INFO ] [DentistServiceImpl:86] Removing dentist...
[2022-12-12 19:54:09] [ INFO ] [Patient:34] Creating patient...
[2022-12-12 19:54:11] [ INFO ] [Patient:41] Searching all the patients...
[2022-12-12 19:54:14] [ INFO ] [Patient:56] Searching patient with id= 1
[2022-12-12 19:54:16] [ INFO ] [Patient:68] Updating all patient data...
[2022-12-12 19:54:18] [ INFO ] [Patient:82] Updating the patient name, lastname and email...
[2022-12-12 19:54:21] [ INFO ] [Patient:56] Searching patient with id= 1
[2022-12-12 19:54:21] [ INFO ] [Patient:92] Removing patient...
[2022-12-12 19:54:28] [ INFO ] [Patient:41] Searching all the patients...
[2022-12-12 19:54:53] [ INFO ] [DentistServiceImpl:30] Creating dentist...
[2022-12-12 19:54:55] [ INFO ] [DentistServiceImpl:48] Searching all the dentists...
[2022-12-12 19:54:56] [ INFO ] [DentistServiceImpl:37] Searching dentist with id= 1
[2022-12-12 19:54:58] [ INFO ] [DentistServiceImpl:62] Updating all dentist data...
[2022-12-12 19:55:00] [ INFO ] [DentistServiceImpl:76] Updating the dentist's first and last name...
[2022-12-12 19:55:02] [ INFO ] [DentistServiceImpl:37] Searching dentist with id= 1
[2022-12-12 19:55:02] [ INFO ] [DentistServiceImpl:86] Removing dentist...
[2022-12-12 19:55:04] [ INFO ] [Patient:34] Creating patient...
[2022-12-12 19:55:07] [ INFO ] [Patient:41] Searching all the patients...
[2022-12-12 19:55:09] [ INFO ] [Patient:56] Searching patient with id= 1
[2022-12-12 19:55:11] [ INFO ] [Patient:68] Updating all patient data...
[2022-12-12 19:55:13] [ INFO ] [Patient:82] Updating the patient name, lastname and email...
[2022-12-12 19:55:15] [ INFO ] [Patient:56] Searching patient with id= 1
[2022-12-12 19:55:18] [ INFO ] [Patient:56] Searching patient with id= 1
[2022-12-12 19:55:18] [ INFO ] [Patient:92] Removing patient...
[2022-12-12 19:55:21] [ INFO ] [Patient:41] Searching all the patients...
[2022-12-12 19:55:37] [ INFO ] [DentistServiceImpl:30] Creating dentist...
[2022-12-12 19:55:38] [ INFO ] [Patient:34] Creating patient...
[2022-12-12 19:55:41] [ INFO ] [TurnService:39] Creating turn...
[2022-12-12 19:55:44] [ INFO ] [TurnService:60] Searching turn with id= 1
[2022-12-12 19:55:46] [ INFO ] [TurnService:49] Searching all the turns...
[2022-12-12 19:55:55] [ INFO ] [TurnService:70] Updating turn data...
[2022-12-12 19:55:57] [ INFO ] [TurnService:60] Searching turn with id= 1
[2022-12-12 19:56:01] [ INFO ] [TurnService:60] Searching turn with id= 1
[2022-12-12 19:56:01] [ INFO ] [TurnService:78] Removing turn...
[2022-12-12 19:58:03] [ INFO ] [DentistServiceImpl:30] Creating dentist...
[2022-12-12 19:58:06] [ INFO ] [DentistServiceImpl:48] Searching all the dentists...
[2022-12-12 19:58:08] [ INFO ] [DentistServiceImpl:37] Searching dentist with id= 1
[2022-12-12 19:58:10] [ INFO ] [DentistServiceImpl:62] Updating all dentist data...
[2022-12-12 19:58:14] [ INFO ] [DentistServiceImpl:76] Updating the dentist's first and last name...
[2022-12-12 19:58:18] [ INFO ] [DentistServiceImpl:37] Searching dentist with id= 1
[2022-12-12 19:58:18] [ INFO ] [DentistServiceImpl:86] Removing dentist...
[2022-12-12 19:58:21] [ INFO ] [Patient:34] Creating patient...
[2022-12-12 19:58:23] [ INFO ] [Patient:41] Searching all the patients...
[2022-12-12 19:58:26] [ INFO ] [Patient:56] Searching patient with id= 1
[2022-12-12 19:58:30] [ INFO ] [Patient:68] Updating all patient data...
[2022-12-12 19:58:33] [ INFO ] [Patient:82] Updating the patient name, lastname and email...
[2022-12-12 19:58:37] [ INFO ] [Patient:56] Searching patient with id= 1
[2022-12-12 19:58:37] [ INFO ] [Patient:92] Removing patient...
[2022-12-12 19:58:40] [ INFO ] [Patient:41] Searching all the patients...
[2022-12-12 19:59:06] [ INFO ] [DentistServiceImpl:30] Creating dentist...
[2022-12-12 19:59:08] [ INFO ] [DentistServiceImpl:48] Searching all the dentists...
[2022-12-12 19:59:10] [ INFO ] [DentistServiceImpl:37] Searching dentist with id= 1
[2022-12-12 19:59:13] [ INFO ] [DentistServiceImpl:62] Updating all dentist data...
[2022-12-12 19:59:16] [ INFO ] [DentistServiceImpl:76] Updating the dentist's first and last name...
[2022-12-12 19:59:51] [ INFO ] [TurnService:39] Creating turn...
[2022-12-12 20:00:09] [ INFO ] [TurnService:49] Searching all the turns...
[2022-12-12 20:00:12] [ INFO ] [TurnService:39] Creating turn...
[2022-12-12 20:00:33] [ INFO ] [DentistServiceImpl:30] Creating dentist...
[2022-12-12 20:00:35] [ INFO ] [DentistServiceImpl:76] Updating the dentist's first and last name...
[2022-12-12 20:00:41] [ INFO ] [DentistServiceImpl:62] Updating all dentist data...
[2022-12-12 20:00:45] [ INFO ] [Patient:34] Creating patient...
[2022-12-12 20:00:50] [ INFO ] [TurnService:39] Creating turn...
[2022-12-12 20:00:57] [ INFO ] [TurnService:60] Searching turn with id= 1
[2022-12-12 20:00:59] [ INFO ] [TurnService:49] Searching all the turns...
[2022-12-12 20:01:00] [ INFO ] [TurnService:70] Updating turn data...
[2022-12-12 20:03:05] [ INFO ] [DentistServiceImpl:48] Searching all the dentists...
